### ISSUE TITLE
TROUBLE SUBMITTING PEER REVIEW#2 FOR NINADH

### ISSUE THREAD

**Issue by divita95** [STAT545-UBC/Discussion-Internal#35]
@ChadFibke @vincenzocoia @Rashedul 
Hi,
I am unable to submit the review for Homework2 for Ninadh as it says that " The repository is empty". Here is the link to repo: https://github.com/STAT545-UBC-students/hw02-Ninadh

And so, I have submitted my review with **only 1 peer review for now**. Kindly look into this and let me know so that I can re-submit the reviews.

Thanks

**Comment by ChadFibke** [comment]
Hey @divita95,

If you are unable to submit a review please randomly pick another student, and submit a review for them instead. Then on canvas just submit the assignment with the link to the 2 reviews you have submitted. 

Hope this helps!
Chad

**Comment by MielleM** [comment]
I've also had the issue of the link to the repo not work, but searching the user's name returning the correct repository-- might be worth a try if people are unable to find the user they're supposed to review.

**Comment by Rashedul** [comment]
It seems to me that @Ninadh did not submit hw02. It is also true for hg03. I can only see her hw01 is present here https://github.com/STAT545-UBC-students/.

@Ninadh do you have any comment?

**Comment by Ninadh** [comment]
@Rashedul @ChadFibke @vincenzocoia

Hi,

Sorry for the trouble. I did submit hw02 but not sure why it is showing empty. I'll try to fix it myself or ask for your help on Tuesday after class. 

I did not submit hw03. I understand that people who are auditing the course are not required to do homeworks as they are not graded. However, participation is important as there is pass/fail. I wrote an email to Dr. Coia today confirming the matter. 

Thanks,
Ninadh
